main.go

```diff
@@ -15,7 +15,7 @@ import (
 	"strings"
 	"text/template"
 
-	"github.com/hashicorp/go-envparse"
+	"github.com/acaloiaro/go-envparse"
```

Comment:
What did you need to override?

Reply:
Hashicorp's go-envparse doesn't support env files that include non-env stuff, like comments, which `ess` requires. 

Under the hood, nothing changed in acaloiaro/go-envparse, but I removed the `replace` in its `go.mod` that renamed it from `github.com/hashicorp/go-envparse => github.com/acaloiaro/go-envparse`. 

I'm assuming that project won't be merging my changes, so I'm making mine I hard fork.